google-cloud-language/src/test/java/com/google/cloud/language/it/ITSystemTest.java

```diff
@@ -47,12 +47,19 @@
 public class ITSystemTest {
 
   private static LanguageServiceClient client;
+  private static String GCS_BUCKET;
+  private static final String GCS_BUCKET_ENV_VAR = "GOOGLE_CLOUD_TESTS_LANGUAGE_BUCKET";
   private static final String ANALYZE_TEXT =
       "Android is a mobile operating system developed by Google, based on the Linux kernel and designed primarily for touchscreen mobile devices such as smartphones and tablets.";
 
   @Before
   public void setUp() throws IOException {
     client = LanguageServiceClient.create();
+    if (System.getenv(GCS_BUCKET_ENV_VAR) != null) {
+      GCS_BUCKET = System.getenv(GCS_BUCKET_ENV_VAR);
+    } else {
+      GCS_BUCKET = "cloud-samples-data";
+    }
```

Comment:
nit: this runs between every test and doesn't need to. We can move this code to a static block outside of the `setUp()` method.

Reply:
Done. Moved it outside of test level setUp method.